runit-java-focal/README.md

```diff
@@ -0,0 +1,56 @@
+socrata/runit-java-focal
+====================
+
+Image based on phusion/baseimage-docker that establises a base set of patterns and tools for building other containers with support for multiple processes via runit. This image is packaged with OpenJDK installed.
+
+### Features
+
+- Ubuntu 20.04 LTS. The base system.
+- A correct init process (my_init) with
+  - proper process reaping
+  - Docker Stop sends SIGTERM to the init process, which stops all processes gracefully on termination.
+- runit	replaces Ubuntu's Upstart. Used for service supervision and management
+- Fixes APT incompatibilities with Docker	See https://github.com/dotcloud/docker/issues/1024.
+- syslog-ng: A syslog daemon is necessary so that many services - including the kernel itself - can correctly log to /var/log/syslog
+- logrotate	Rotates and compresses logs on a regular basis.
+- SSH server: disabled by default
+- cron	The cron daemon must be running for cron jobs to work.
+- setuser	A tool for running a command as another user. Easier to use than su, has a smaller attack vector than sudo, and unlike chpst this tool sets $HOME correctly. Available as /sbin/setuser.
+
+See the usage section below
+
```

Comment:
Just to prevent copypasta creep, do we really need to include notes about envconsul, ark-host, env_parse in language-level images? They might be mentioned in the runit base images, but repeating the info here seems redundant if not misleading.

Reply:
After a closer look, I think that readme was originally copied from a the runit base image, I'll make some changes